testcases/InstallUpstreamKernel.py

```diff
@@ -74,7 +74,7 @@ def is_url(path):
             if urlparse(path).scheme in valid_schemes:
                 return True
             return False
-
+        self.cv_SYSTEM.goto_state(OpSystemState.OFF)
```

Comment:
@abdhaleegit do you see any issues without this change?, so far I have not noticed, better to go as separate commit with issue mentioned if that case

Reply:
@abdhaleegit Even I dont see any requirment for  this line. Please let me know why this is requried.

Reply:
@sathnaga @harihare In a CI environment, user expects system to be back to a normal state after every build and test

so to have a clean system for next run we need a reboot  and OpSystemState.OFF  is necessary if the kernel has crashed or system is hung or systems ssh is down  or console is stuck.. also  due to tests most of the prestored configurations are altered and it requires reboot to restore the required configurations